### PR TITLE
chore: Add PR template

### PR DESCRIPTION
**Pull request by alainncls**
## What does this PR do?

Adds a PR template (like the one used right here).

### Type of change

- [X] Chore
- [ ] Bug fix
- [ ] New feature
- [ ] Documentation update

### CODE REVIEW

**Review by satyajeetkolhapure — APPROVED** [review]

**Review by dawnseeker8 — APPROVED** [review]
approved.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## What does this PR do?
```
Comment (Mrtenz) [inline .github/PULL_REQUEST_TEMPLATE.md:1]:
```suggestion
## Description
```

**Review by Mrtenz — APPROVED** [review]
I assume this will be removed before upstreaming

**Comment by alainncls** [comment]
> I assume this will be removed before upstreaming

Why? I mean, isn't it useful?

**Comment by Mrtenz** [comment]
> > I assume this will be removed before upstreaming
> 
> Why? I mean, isn't it useful?

We just don't really use these in the Snaps team. 😅

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**